worldengine/cli/main.py

```diff
@@ -95,7 +96,7 @@ def generate_plates(seed, world_name, output_dir, width, height,
                                                    num_plates=num_plates)
 
     world = World(world_name, width, height, seed, num_plates, -1.0, "plates")
-    world.set_elevation(array_to_matrix(elevation, width, height), None)
+    world.set_elevation(numpy.array(array_to_matrix(elevation, width, height)), None)
     world.set_plates(array_to_matrix(plates, width, height))
```

Comment:
Since you flattened 2d array into 1d array with numpy, I know that numpy can take a 1d array and `numpy.reshape` into an Xd array.  Can you replace this function too please? :)